README.md

```diff
@@ -50,6 +50,13 @@ run: `npm install` to download all dependencies.
 run: `npm start` to start the live server. Note: must be running on localhost:3000
 g
 
+# Map setup
+
+Setting up map for dashboard:
+
+Must have a token setup in a .env.local file (must create file on fresh clone) with the following format for map to render
+` REACT_APP_MAPBOX_TOKEN = "Token string" `
+
```

Comment:
Really good in adding documentation on the process of adding the react map token

Reply:
@ashtilawat23 If this requires a change to the .env file I think we also need a corresponding entry added to the deployed site.